src/style.css

```
@@ -1,3 +1,96 @@
-body {
+:root {
+  --accent-color: #00acdc;
+}
+
+body,
+ul {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+img {
+  width: 40px;
+  height: 40px;
+}
+
+header {
```

Use only classes selector, fix it everywhere
```suggestion
.header {
```